### ISSUE TITLE
Way to create untyped column?

### ISSUE THREAD

**Issue by schveiguy** [groue/GRDB.swift#169]
SQLite supports columns without a type:

```SQL
CREATE TABLE example (column1 INT, column2)
```

However, with GRDB, the table.column function only allows creating columns with a given type:

```swift
            db.create(table: "example") { t in
                t.column("column1", .integer)
                t.column("column2") // error
            }
```

is there a way to do this?

**Comment by groue** [comment]
Hello @schveiguy, glad to see you again!

According to [SQLIte doculemtation](https://www.sqlite.org/datatype3.html#determination_of_column_affinity), an untyped column has the exact same behavior as a numeric column:

```swift
db.create(table: "example") { t in
    t.column("column1", .integer)
    t.column("column2", .numeric)
}
```

This will make your code compile, and your database 100% equivalent (but for the *syntax* of the table declaration).

**Comment by schveiguy** [comment]
Thanks, that looks correct! I will go ahead with this (was just using a create statement, but was sure there was a better way)

**Comment by groue** [comment]
Support for untyped columns has landed: 5f2d6f8e2f97e7dc79a741901ef4faf2155c3697 in [development branch](https://github.com/groue/GRDB.swift/tree/development).

**Comment by groue** [comment]
Shipped in [v1.2](https://github.com/groue/GRDB.swift/blob/master/CHANGELOG.md#12)
